tools/templates/channel_wrappers.tpl

```diff
@@ -0,0 +1,69 @@
+# This file auto-generated by `generate_schema_interface.py`.
+# Do not modify this file directly.
+
+import traitlets as T
+import pandas as pd
+
+from ...utils import parse_shorthand, infer_vegalite_type
+from ...utils import INV_TYPECODE_MAP, TYPE_ABBR
+
+from .._interface import Type
+
+{%- for object in objects %}{% for import in object.imports %}
+{%- set comma = joiner(", ") %}
+from {{ import.module }} import {% for name in import.names %}{{ comma() }}{{ name }}{% endfor %}
+{%- endfor %}{% endfor %}
+
+{% for object in objects %}
+class {{ object.name }}({{ object.base.name }}):
+    """Wrapper for Vega-Lite {{ object.base.name }} definition.
+    {{ object.base.description }}
+    Attributes
+    ----------
+    shorthand: Unicode
+        A shorthand description of the channel
+    {% for attr in object.base.attributes -%}
```

Comment:
We should remove the blank line only if the attribute contains no description. I don't know how to do that.

A better solution would be to do a PR to the vega-lite schema to fill-in missing descriptions.